modules/skybrowser/skybrowsermodule_lua.inl

```diff
@@ -0,0 +1,589 @@
+        for (std::unique_ptr<TargetBrowserPair>& pair : pairs) {
+            std::string id = pair->browserId();
+            glm::ivec3 color = pair->borderColor();
+            std::string script = fmt::format(
+                "openspace.skybrowser.setBorderColor('{}', {}, {}, {})", 
+                id, 
+                color.r, 
+                color.g, 
+                color.b
+            );
+            global::scriptEngine->queueScript(
+                script,
+                scripting::ScriptEngine::RemoteScripting::Yes
+            );
+        }
+    }
+   
+    // To ensure each node in a cluster calls its own instance of the wwt application
+    // Do not send this script to the other nodes
+    global::scriptEngine->queueScript(
+        "openspace.skybrowser.sendOutIdsToBrowsers();",
+        scripting::ScriptEngine::RemoteScripting::No
+    );
+    
+    return 0;
+}
+
+int sendOutIdsToBrowsers(lua_State* L) {
+    // This is called when the sky_browser website is connected to OpenSpace
+    ghoul::lua::checkArgumentsAndThrow(L, 0, "lua::sendOutIdsToBrowsers");
+
+    // Send out identifiers to the browsers
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+    std::vector<std::unique_ptr<TargetBrowserPair>>& pairs = module->getPairs();
+    for (std::unique_ptr<TargetBrowserPair>& pair : pairs) {
+        pair->sendIdToBrowser();
+    }
+
+    return 0;
+}
+
+int initializeBrowser(lua_State* L) {
+    // Initialize browser with ID and its corresponding target
+    ghoul::lua::checkArgumentsAndThrow(L, 1, "lua::initializeBrowser");
+    const std::string id = ghoul::lua::value<std::string>(L, 1);
+
+    LINFO("Initializing sky browser " + id);
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+    TargetBrowserPair* found = module->getPair(id);
+    if (found) {
+        found->setIsSyncedWithWwt(true);
+        found->initialize();
+    }
+
+    return 0;
+}
+    
+int addPairToSkyBrowserModule(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 2, "lua::addPairToSkyBrowserModule");
+    auto [targetId, browserId] = ghoul::lua::values<std::string, std::string>(L);
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+
+    LINFO("Add browser " + browserId + " to sky browser module");
+    LINFO("Add target " + targetId + " to sky browser module");
+        
+    module->addTargetBrowserPair(targetId, browserId);
+
+    return 0;
+}
+
+int getListOfImages(lua_State* L) {
+    // Send image list to GUI
+    ghoul::lua::checkArgumentsAndThrow(L, 0, "lua::getListOfImages");
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+
+    // If no data has been loaded yet, download the data from the web!
+
+    if (module->nLoadedImages() == 0) {
+        std::string root = "https://raw.githubusercontent.com/WorldWideTelescope/"
```

Comment:
should this URL really be hardcoded like this? 🤔 If it would need to change for some reason, it would be very hard for the user to do